library/Fs.re

```diff
@@ -40,7 +38,7 @@ let exists = path => {
   };
 };
 
-let realpath = Filename.realpath;
+let realpath = s => Path.absoluteExn(s) |> Path.toString;
```

Comment:
Make sure this is the same thing, this will throw if it's not a path string

Reply:
This is not what `realpath` does, and I believe this is what causing the build errors.
`realpath` is like `Unix.readlink`, but it doesn't throw if it can't reach the destination: http://man7.org/linux/man-pages/man1/realpath.1.html

Reply:
Maybe I can make it better in another PR - assume `system` version if there's an error reading the link to the node version (because I use `/dev/null/installation` as the target 😜)